### PR TITLE
Update Dart dependency to include fixes in x64 Dart precompilation

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/engine/core/script/dart_init.cc [hunk 1]
```diff
@@ -86,10 +86,6 @@ static const char* kDartCheckedModeArgs[] = {
     "--error_on_bad_override",
 };
 
-void UnhandledExceptionCallback(Dart_Handle error) {
-  LOG(ERROR) << Dart_GetError(error);
-}
```
Comment (abarth) [inline sky/engine/core/script/dart_init.cc]: Did you intend to remove this?

**Comment by abarth** [comment]
LGTM but note question above.

File: sky/engine/core/script/dart_init.cc (hunk 1 above)
Comment (chinmaygarde) [inline sky/engine/core/script/dart_init.cc]: Yes. The Dart API no longer supports the embedder providing this argument via Dart_Initialize. Will follow up on if there is now an alternative. Basically, that argument has to be nil. See https://github.com/dart-lang/sdk/blob/master/runtime/vm/dart_api_impl.cc#L1131

**Comment by chinmaygarde** [comment]
After following up on this, it looks like this mechanism was entirely removed. There is no alternative.

**Pull request merged**